### ISSUE TITLE
Link toolbar keyboard accessibility

### ISSUE THREAD

**Issue by Acrophost** [ckeditor/ckeditor5#12333]
## 📝 Provide a description of the improvement

Currently link toolbar is not accessible via keyboard - overlooked while making/testing [#12332](https://github.com/ckeditor/ckeditor5/pull/12064) :(

https://user-images.githubusercontent.com/66953380/186476300-c6c629ff-6c32-4810-8244-214a4721b615.mov

---

If you'd like to see this improvement implemented, add a 👍 reaction to this post.

**Comment by oleq** [comment]
It's by design because otherwise, if the focus moved right away to the balloon on click, you wouldn't be able to type more text if you moved caret into the link by using mouse. You can hit Tab to focus the balloon instead.

**Comment by Acrophost** [comment]
@oleq yes, but you still have to click the link for the toolbar to even show. The balloon doesn't appear when you go into/select the link with keyboard and neither ALT+F10 nor tab is helping.

**Comment by oleq** [comment]
You can use Ctrl+K for the link UI to show up. I know this could be misleading because it looks like a toolbar. WDYT @mlewand ?

**Comment by Reinmar** [comment]
I had the same observation. I wonder how many other pieces of our UI (other contextual balloons) would benefit (or maybe not) from being reachable in the same way?

Or in other words: should we differentiate between toolbars and other contextual panels?

**Comment by oleq** [comment]
I suppose the right person to answer this question is @Comandeer. 

To me, the link balloon is a sort of a hybrid: the first one that shows up allows you to open the link and get to the editing tools and could be considered a toolbar. The other one with the URL input is definitely not a toolbar.

**Comment by Comandeer** [comment]
IMO if it quacks like a duck, it's a duck. I don't see a reason to treat something that looks like a balloon toolbar in a different way.

For the user it would be counterintuitive to have two ways of interacting with a contextual panel:

*   some of them will be shown on selection, some of them only on some keystroke or mouse click/touch tap,
*   some of them will be focusable via toolbar cycling keystroke (<kbd>Alt</kbd>+<kbd>F10</kbd>), some of them will react to <kbd>Tab</kbd>.

Due to that, the user will need to memorize which ones use which method – as the differences are not obvious from the UI.
